tensorflow/compiler/xla/backends/profiler/gpu/device_tracer_rocm.cc

```diff
@@ -1101,11 +1101,6 @@ Status GpuTracer::Stop() {
   return OkStatus();
 }
 
-Status GpuTracer::DoCollectData(XSpace* space) {
```

Comment:
Please also remove the declaration in line 950 when removing this member function definition.